spec/helpers/comments_helper_spec.rb

```diff
@@ -60,5 +60,24 @@
         expect(helper.css_classes_for_comment(unreviewed_comment)).to eq("unreviewed comment group user-#{user.id}")
       end
     end
+
+    context "when comment author is an official" do
```

Comment:
We can say "commenter" instead of "comment author" in these descriptions for brevity and consistency with the other tests.